api/src/main/java/net/kyori/adventure/sound/SoundStop.java

```diff
@@ -32,7 +32,7 @@
 import static java.util.Objects.requireNonNull;
 
 /**
- * A sound stop.
+ * A packet to stop any in-game sounds which can be sent to players.
```

Comment:
Do not mention "packet" here.

Reply:
Do you have a suggestion on how to reword this? I can't think of anything

Reply:
`A combination of a sound and sound source, used for stopping in-game sounds that are being played on a game client.`?

Reply:
But a `SoundStop` can be a sound or a source by itself? Is a "and/or" fine?

Reply:
it can be either, but it should be clear that the client will only stop sounds that meet *all* of the specified parameters.